opencti-platform/opencti-front/src/private/components/workspaces/WorkspaceCreation.jsx

```diff
@@ -74,6 +74,7 @@ const WorkspaceCreation = ({ paginationOptions, type }) => {
     commitImportMutation({
       variables: { file: importedFile },
       onCompleted: (data) => {
+        inputRef.current.value = ''; // Reset the input uploader ref
```

Comment:
You should put that in an updater in case of an error.
Plus I would set to null instead of '', at the input is a file input (not really important)